Library/Formula/ola.rb

```diff
@@ -2,8 +2,8 @@
 
 class Ola < Formula
   homepage "http://www.openlighting.org/ola/"
-  url "https://github.com/OpenLightingProject/ola/releases/download/0.9.3/ola-0.9.3.tar.gz"
-  sha1 "f6a81087761218063a4bb8006b73ffa407cd0170"
+  url "https://github.com/OpenLightingProject/ola/releases/download/0.9.5/ola-0.9.5.tar.gz"
+  sha1 "e9b0d8e4e3e4acd696cd7a0877d72d593727a7bc"
```

Comment:
That is no problem! I can do that, I just was unaware how to calculate it (I did some research and came up with the same value you did.)

So, in addition to changing the value, I should change the key "sha1" to be "sha256", I expect?